repo/packages/covfie/package.py

```diff
@@ -0,0 +1,29 @@
+from spack.package import *
+
+
+class Covfie(CMakePackage):
+    homepage = "https://github.com/acts-project/covfie"
+    git = "https://github.com/acts-project/covfie.git"
+    list_url = "https://github.com/acts-project/covfie/tags"
+
+    version("0.5.0", commit="59bd2e0a4861db6e5898606e0167da895505e46b")
+
+    variant("cuda", default=False, description="Enables the CUDA platform")
+
+    depends_on("cmake@3.17:", type="build")
+
+    def cmake_args(self):
+        def cmake_bool(p):
+            return "On" if p else "Off"
+
+        spec = self.spec
+
+        args = [
+            "-DCMAKE_BUILD_TYPE=Release",
+            "-DCOVFIE_PLATFORM_CPU=On",
+            "-DCOVFIE_PLATFORM_CPU={}".format(cmake_bool(spec.satisfies("+cuda"))),
```

Comment:
Probably this package is too small to justify a dedicated utility function, but consider using the upstream utility [self.define_from_variant()](https://spack.readthedocs.io/en/latest/spack.build_systems.html#spack.build_systems.cmake.CMakeBuilder.define_from_variant)